modules/govuk_jenkins/manifests/jobs/bouncer_cdn.pp

```diff
@@ -28,8 +28,13 @@
 
   $check_name = 'bouncer-cdn-configuration'
 
-  # FIXME: go back to using $app_domain once we have a single Icinga instance in AWS
-  $job_url = "https://deploy.${::aws_environment}.govuk.digital/job/Bouncer_CDN/"
+  if $::aws_migration and ($::aws_environment != 'integration') {
+    $hosting_env_domain = "blue.${::aws_environment}.govuk.digital"
+  }
+  else {
+    $hosting_env_domain = $app_domain
+  }
+  $job_url = "https://deploy.${hosting_env_domain}/job/Bouncer_CDN/"
```

Comment:
Does this job also exist in Carrenza? Bouncer just runs in AWS, so even if it does exist in Carrenza, maybe removing the job would be an option?

Reply:
Are you suggesting amending to something like the following, where `$job_url` is undefined in Carrenza?

```
if $::aws_migration and ($::aws_environment != 'integration') {
    $job_url = "https://deploy.${hosting_env_domain}/job/Bouncer_CDN/"
}
```